pkg/users/local.js

```diff
@@ -271,6 +271,35 @@ function is_user_in_group(user, group) {
     return false;
 }
 
+function change_shell(user, shell) {
+    var dfd = cockpit.defer();
+
+    cockpit.spawn([ "chsh", "-s", shell, user ], {superuser: "require", err: "out" })
+        .done(function() {
+            dfd.resolve();
+        })
+        .fail(function(ex, response) {
+            if (ex.exit_status) {
+                console.log(ex);
+                if (response)
+                    ex = new Error(response);
+                else
+                    ex = new Error(_("Failed to change password"));
+            }
+            dfd.reject(ex);
+        });
+
+    return dfd.promise();
+}
+
+function turn_on_session_recording(user) {
+    return change_shell(user, "/usr/bin/tlog-rec-session");
+}
+
+function turn_off_session_recording(user) {
+    return change_shell(user, "/bin/bash");
```

Comment:
This might not be true - can the original value be remembered somewhere? Also, don't you support different shells with session recording? I thought this was just a wrapper, so coudl the shell be something like `tlog-rec-session /bin/bash`?